pkg/proxy/handlers/spacelister_get.go

```diff
@@ -26,6 +28,8 @@ func HandleSpaceGetRequest(spaceLister *SpaceLister, GetMembersFunc cluster.GetM
 	// get specific workspace
 	return func(ctx echo.Context) error {
 		requestReceivedTime := ctx.Get(regsercontext.RequestReceivedTime).(time.Time)
+		publicViewerEnabled := configuration.GetRegistrationServiceConfig().PublicViewerEnabled()
+		ctx.Set(context.PublicViewerEnabled, publicViewerEnabled)
```

Comment:
getting from configuration when needed could lead you to fetch two different values for the same request. This can happen if the configuration is updated in the middle of processing a request. Very unlikely, but still possible.

Also, having this in context allows us to easily log the value.